configuration-api/main/mongo/tenantsQueries.js

```diff
@@ -18,7 +18,7 @@ const Config = connection.model('TenantConfig', TenantConfig);
 
 async function get(data) {
   const tenants = await Config.find({ name: { $in: data } });
-  if (tenants.length === data.length) {
+  if (tenants.length >= data.length) {
     return await tenants;
```

Comment:
> Isn't `await` redundant here if `tenants` is just an array of objects? or is it something else?

the await is necessary because the Mongose api's are async otherwise a pending promises will be returned instead of the tenants values